api/optimeet/activities/views.py

```diff
@@ -0,0 +1,20 @@
+from django.shortcuts import render
+from django.http import HttpResponse
+from rest_framework.response import Response
+from rest_framework.decorators import api_view
+from django.views.decorators.csrf import csrf_exempt
+
+import json
+import os
+
+@csrf_exempt
+@api_view(['GET'])
+def get_activities(request):
+    
+    here = os.path.dirname(os.path.abspath(__file__))
+    
+    with open(os.path.join(here, 'Activities.json')) as file:
+        
+        activities = json.load(file)
```

Comment:
Can we move this outside of the get function so that it only gets loaded once when the file loads then simply returns it whenever the GET request is made?

Reply:
done